bin/root-transition.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+# Copyright 2019 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script extends the lifetime of the self-signed Citadel root certificate in the current cluster.
+# It offers the following options:
+# - check: Check the current root certificate lifetime.
+# - transition: Extend the lifetime of the current root certificate.
+# - verify: Check the new workload certificates are generated.
+
```

Comment:
mention that this script required kubectl, openssl and `bc`.

I did not have `bc` in mycluster so the script failed.

Reply:
Done. Thanks.